src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added the list of constraint violations' parameters in `ConstraintViolationListNormalizer`
+ * added a `deep_object_to_populate` context option to recursive denormalize on `object_to_populate` object.
```

Comment:
Cannot it be the default?

Reply:
Would it be a BC break?

Reply:
The current behavior is to not populate the children object. So if we enable it, this is going to be a BC break.
So I think we could enable it by default in Symfony 5.

Reply:
Cannot be change in 5 either as we need an upgrade path from 4 to 5, which is going to be impossible.